modules/administration-guide/partials/snip_creating-che-database.adoc

```diff
@@ -0,0 +1,6 @@
+. Create {prod-short} database:
```

Comment:
Does this snip makes sense if we use it only in one place?
Are we going to reuse it somewhere?

Reply:
Yes, I am going to reuse them in other PRs